Jenkinsfiles/standby-push.groovy

```diff
@@ -31,6 +31,13 @@ stage('Push') {
 }
 
 stage('Push EKS') {
+
+    // add additional notify
+    notify_slack([
+        status: "Pushing to EKS",
+        message: "${repo} image ${tag}"
+    ])
```

Comment:
I don't think this will work as it is. First of all, you'd need to use `utils.notify_slack`, but also `repo` and `tag` are undefined. I think it'd be better to use `utils.announce_push("EKS")` here and then also change the `announce_push` function within `Jenkinsfiles/utils.groovy` to something like:
```groovy
def announce_push(target=null) {
    /*
     * Announce the push.
     */
    if (target == null) {
        target = get_target_name()
    }
    def repo = get_repo_name()
    def tag = get_commit_tag()
    notify_slack([
        status: "Pushing to ${target}",
        message: "${repo} image ${tag}"
    ])
}
```

Reply:
Oh good catch, I took a slightly different approach with this and instead just modified the `get_target_name` function instead to accept an argument. See commit bb604699920d28b2fe39cede91c6fefd23a3e34b for details